### PR TITLE
Add google autocomplete.

### PR DESCRIPTION
**Pull request by wsite**

### CODE REVIEW

File: client/static/app/modules/calendar/services/calendar.service.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ define(function(require) {
...
       $http({
-        method: 'GET',
+        method: 'POST',
         url: '/api/plays/calendar',
-        params: {
+        data: {
           startDate: start.format('DD-MM-YYYY'),
           endDate: end.format('DD-MM-YYYY'),
-          city: city
+          cityId: (city) ? city.id : null
```
Comment (andrewMuntyan) [inline client/static/app/modules/calendar/services/calendar.service.js]: why (city)? for what did you need brackets?

File: server/fixtures/index.js [hunk 2]
```diff
@@ -7,12 +7,9 @@ var config = require('../config'),
 mongoose.connect(config.mongo.uri, config.mongo.options);
 
 var defer = Q.defer();
-fixtures(require('./cities'), mongoose, function() {
-  defer.resolve();
-});
 
 defer.promise.then(function() {
-  console.log('Cities loaded');
+  console.log('Done');
```
Comment (andrewMuntyan) [inline server/fixtures/index.js]: ?

File: client/static/app/modules/calendar/services/calendar.service.js (hunk 1 above)
Comment (wsite) [inline client/static/app/modules/calendar/services/calendar.service.js]: it look better with them

File: client/static/assets/scss/plays.scss [hunk 3]
```diff
@@ -22,3 +22,6 @@
 .players-widget {
   margin-top: 15px;
 }
+.pac-container{
+  z-index: 9999;
```
Comment (artyomtrityak) [inline client/static/assets/scss/plays.scss]: ?

File: server/modules/plays/index.js [hunk 4]
```diff
@@ -10,13 +10,12 @@ var express = require('express'),
 
 // Registering routes
 router.get('/', controller.getPlays);
-router.get('/calendar', controller.getPlays);
+router.post('/calendar', controller.getPlays);
```
Comment (artyomtrityak) [inline server/modules/plays/index.js]: why post?

**Comment by artyomtrityak** [comment]
looks good, merge it

File: server/modules/plays/index.js (hunk 4 above)
Comment (wsite) [inline server/modules/plays/index.js]: it look better to send in post data obj instead of get

File: client/static/assets/scss/plays.scss (hunk 3 above)
Comment (wsite) [inline client/static/assets/scss/plays.scss]: modal window have zindex 1200 so i put 9999 to overwrite it

File: server/fixtures/index.js (hunk 2 above)
Comment (wsite) [inline server/fixtures/index.js]: We dont need city any more

**Pull request merged**